kratos/modeler/operation/compute_surrogate_boundary_data.cpp

```diff
@@ -0,0 +1,165 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortes Danes
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/parallel_utilities.h"
+#include "compute_surrogate_boundary_data.h"
+
+namespace Kratos {
+
+Parameters ComputeSurrogateBoundaryData::GetDefaultParameters() const
+{
+    return Parameters(R"({
+        "type" : "compute_surrogate_boundary_data",
+        "model_part_name": "Undefined",
```

Comment:
```suggestion
        "model_part_name": "",
```
We normally leave this one empty as default. Note there's a chance that an evil mind names a model part as "Undefined".